contracts/modules/royalty/policies/IpRoyaltyVault.sol

```diff
@@ -114,7 +114,6 @@ contract IpRoyaltyVault is IIpRoyaltyVault, ERC20SnapshotUpgradeable, Reentrancy
         _mint(rtReceiver, supply);
```

Comment:
we may not need this line as variable was deprecated:

 `$.lastSnapshotTimestamp = uint40(block.timestamp);`

Reply:
removed the line.